institut-teknologi-bandung-tesis-magister.csl

```diff
@@ -13,14 +13,18 @@
       <email>hendy@hendyirawan.com</email>
       <name>Hendy Irawan</name>
     </author>
+    <author>
+      <email>firdan.machda@gmail.com</email>
+      <name>Firdan Machda</name>
+    </author>
     <category citation-format="author-date"/>
     <category field="engineering"/>
     <category field="generic-base"/>
-    <summary>Sekolah Pascasarjana Institut Teknologi Bandung - Tesis Magister</summary>
-    <updated>2016-07-24T00:59:00+07:00</updated>
+    <summary>perbaikan dari csl Sekolah Pascasarjana Institut Teknologi Bandung - Tesis Magister oleh Hendy karena beberapa tidak sesuai dengan pedoman</summary>
+    <updated>2016-07-27</updated>
```

Comment:
The automated checks are failing because the "updated" timestamp must be valid according to the xsd:dateTime syntax (see http://books.xmlschemata.org/relaxng/ch19-77049.html).

E.g. `<updated>2016-07-27T00:00:00</updated>` would be valid